### PR TITLE
Fix auto refresh v2

### PR DESCRIPTION
**Pull request by ruowan**

### CODE REVIEW

File: src/test/docs-test.ts [hunk 1]
```diff
@@ -73,4 +73,49 @@ describe('docs test getSwaggerFiles', () => {
 
     assert.deepStrictEqual(res.stable, ['src/test/readmes/Microsoft.Kusto/stable/2022-07-07/kusto.json'])
   })
+
+  it.skip('Test getSwaggerFiles for api management.', () => {
```
Comment (live1206) [inline src/test/docs-test.ts:77]: why is this skipped?

File: src/test/docs-test.ts [hunk 2]
```diff
@@ -73,4 +73,49 @@ describe('docs test getSwaggerFiles', () => {
 
     assert.deepStrictEqual(res.stable, ['src/test/readmes/Microsoft.Kusto/stable/2022-07-07/kusto.json'])
   })
+
```
Comment (live1206) [inline src/test/docs-test.ts:76]: Add a test for non-existing readme file?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # Changelog
 
+## 0.8.14
+
+- Filter out `only` postfix tag for getSwaggerFiles function
```
Comment (live1206) [inline CHANGELOG.md:5]: why is this?

Comment (live1206) [inline CHANGELOG.md:5]:
```suggestion
- Filter out `only` suffix tag for getSwaggerFiles function
```

File: src/test/docs-test.ts (hunk 1 above)
Comment (ruowan) [reply src/test/docs-test.ts:77]: That's for local test. When we want to test it with some local spec repo.

File: CHANGELOG.md (hunk 3 above)
Comment (ruowan) [reply CHANGELOG.md]: Some only tag doesn't contains all swagger files. That only suffix tag is  for some special purpose.

**Review by live1206 — APPROVED** [review]

**Review by tianxchen-ms — APPROVED** [review]

**Pull request merged**